docker-agent-installed/alpine/Dockerfile

```diff
@@ -25,8 +25,8 @@ FROM base AS final
 # 
 # the agent is installed at /usr/local/new-relic-dotnet-agent
 RUN apk update && apk add --no-cache wget tar \
-    && wget https://download.newrelic.com/dot_net_agent/latest_release -A '*amd64.tar.gz' -r \
-    && tar -xzf download.newrelic.com/dot_net_agent/latest_release/*.tar.gz -C /usr/local \ 
+    && wget https://download.newrelic.com/dot_net_agent/latest_release -A '*.*amd64.tar.gz' -r \
+    && tar -xzf download.newrelic.com/dot_net_agent/latest_release/*.*amd64.tar.gz -C /usr/local \ 
```

Comment:
```suggestion
    && tar -xzf download.newrelic.com/dot_net_agent/latest_release/newrelic-dotnet-agent_arm64.tar.gz -C /usr/local \ 
```

Reply:
I think that this was the wrong suggestion. The previous command just downloaded the amd64 binaries, and this recommendation is attempting to unpack the arm64 binaries. Both commands should probably refer to the same platform type.

Reply:
Oops! Copy-pasta got me!